kbuild/v2/scripts/lib.sh

```diff
@@ -33,3 +33,30 @@ get_deb_version() {
     deb_version=${deb_version##*_}
     echo -n $deb_version
 }
+
+upload_artifact() {
+    local archive="$1"
+    local astore_path="$2"
+    local arch="$3"
+    local tag="$4"
+    local public="$5"
+
+    if [ ! -r "$archive" ] ; then
+        echo "ERROR: unable to find archive: $archive"
+        exit 1
+    fi
+
+    # upload archive to astore
+    enkit astore upload "${archive}@${astore_path}" -a $arch -t "$tag"
+
+    if [ "$public" = "private" ] ; then
```

Comment:
public / private business can go away once @rajachan changes are in?

Reply:
It looks like those changes are in.  So everything can be made private.

Reply:
@ccontavalli -- I disabled tagging and the public/private stuff.  PTAL.